tests/test_core.py

```diff
@@ -4354,6 +4354,42 @@ def test_dylink_hyper_dupe(self):
       full = self.run_js('src.js')
       self.assertContained("warning: symbol '_sideg' from '%s' already exists" % libname, full)
 
+  @needs_dlfcn
+  def test_dylink_load_compiled_side_module(self):
+    self.set_setting('FORCE_FILESYSTEM')
+    self.emcc_args.append('-lnodefs.js')
+    self.set_setting('INITIAL_MEMORY', '64mb')
+    if not self.has_changed_setting('ASSERTIONS'):
+      self.set_setting('ASSERTIONS', 2)
```

Comment:
We should let the differrent test run configuration determine if we are using ASSERTIONS or no?   Maybe just remove these two lines?